analysis/question_1/top5_countries_suicide_rate.py

```diff
@@ -0,0 +1,43 @@
+# This code is the analysis on the top 5 countries with hight suicide rates
+# It further attempts to explore their differences and similarity of suicide rates in different age groups
+# barplot from Seaborn library is used for visualisation 
+
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+# Getting Data of crude suicide rate
+suicide_rate_data_filepath = os.path.join("..\\processed_data", "crude_suicide_rates.csv")
+suicide_rate_dataframe = pd.read_csv(suicide_rate_data_filepath, index_col=0)
+#print(human_resource_dataframe.head())
+
+
+# Calculate the total suicide rate of all different ages
+suicide_rate_dataframe['all_age'] = suicide_rate_dataframe['80_above']+ suicide_rate_dataframe['70to79'] + suicide_rate_dataframe['60to69']+ suicide_rate_dataframe['50to59']
```

Comment:
all_age suicide rate figures may not be correct without considering the proportion of each age group in the countries population
For example , "80_above"'s population is 100 with 80% suicide rate, "0to80"'s population is 900 with suicide rate 1%, the suicide rate should be (100x50% + 900*1%)/(100+900)*100% = 5.9%

Reply:
Hi Ming,
According to the data information [here](https://www.kaggle.com/twinkle0705/mental-health-and-suicide-rates?select=Crude+suicide+rates.csv) , the suicide rates of different age groups are counted based on the general population of 100,000. So the rate in different age groups are **mutually exclusive** and it is safe to add up. So, this can be made as our assumption.